README.md

```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
 result = sma.evaluate(evset)
 ```
 
+Check the [Getting Started tutorial](examples/tutorials/getting_started.ipynb) to try it!
+
 ## Key features
 
 These are what set Temporian apart.
 
 - **Simple and powerful API**: Temporian exports high level operations making processing complex programs short and ready to read.
+- **Flexible data model**: Temporian models temporal data as a sequence of events, supporting non-uniform sampling timestamps seamlessly.
 - **Prevents modeling errors**: Temporian programs are guaranteed not to have future leakage unless the user calls the `leak` function, ensuring that models are not trained on future data.
 - **Iterative development**: Temporian can be used to develop preprocessing pipelines in Colab or local notebooks, allowing users to visualize results each step of the way to identify and correct errors early on.
 - **Efficient and well-tested implementations**: Temporian contains efficient and well-tested implementations of a variety of temporal data processing functions. For instance, our implementation of window operators is **x2000** faster than the same function implemented with NumPy.
 - **Wide range of preprocessing functions**: Temporian contains a wide range of preprocessing functions, including moving window operations, lagging, calendar features, arithmetic operations, index manipulation and propagation, resampling, and more. For a full list of the available operators, see the [operators documentation](https://temporian.readthedocs.io/en/latest/reference/temporian/core/operators/).
 
-## Documentation
+## Why Temporian?
```

Comment:
I'd kill this section, especially since the M5 notebook isn't really readable atm

Reply:
Agree. Removed 😬